src/basho_bench_config.erl

```diff
@@ -28,21 +28,38 @@
 -compile(export_all).
 -endif.
 
--export([load/1,
-         normalize_ips/2,
-         set/2,
-         get/1, get/2]).
+-export([
+    load/1,
+    normalize_ips/2,
+    set/2,
+    get/1,
+    get/2,
+    set_local_config/1
+]).
 
 -export([start_link/0]).
 
 % Gen server callbacks
--export([code_change/3, init/1, terminate/2, handle_call/3, handle_cast/2, handle_info/2]).
+-export([
+    code_change/3,
+    init/1,
+    terminate/2,
+    handle_call/3,
+    handle_cast/2,
+    handle_info/2
+]).
+
 
 -include("basho_bench.hrl").
 
--record(basho_bench_config_state, {}).
```

Comment:
That was part of @chewbranca's cleanup changes before he waded into rearranging my prototype code.